content/templates/profile.md

```diff
@@ -1,13 +1,20 @@
 ---
-created: 2023-11-17
-title: 
+created: 2023-11-30
```

Comment:
you want {{DATE}} - this will auto-convert to today's date when you insert the template.